lib/HttpClient.js

```diff
@@ -368,6 +368,7 @@ function rawRequest(opts, cb) {
         if (_socket.writable && !_socket._connecting) {
             startRequestTimeout();
             clearTimeout(connectionTimer);
+            cbCalled = true;
```

Comment:
Should this be guarded by `if (cbCalled === false)` too?

Reply:
Bummer. I guess we are back to where we started with the original once code (without strict). I think what I will do then, is document where cb can be called, and document the (known) scenarios in which cb can be called more than once, and then begrudgingly go back to using once. :)